packages/lib/src/validations/index.ts

```diff
@@ -100,22 +100,6 @@ function validateStorage(
   };
 }
 
-function tryGetUninitializedBaseContracts(contract: Contract): string[] {
```

Comment:
Should we remove `packages/lib/src/validations/Initializers.ts` as well then, if we are removing this whole function?

Reply:
Certainly. SDK is hard enough to navigate even without dead code.